lit_gpt/utils.py

```diff
@@ -371,6 +373,20 @@ def __iter__(self) -> Self:
         return self
 
 
+def copy_config_files(source_dir: Path, out_dir: Path) -> None:
+    """Copies the specified configuration and tokenizer files into the output directory."""
+
+    # TODO: All the config files need to be present, should we raise an error if one is missing?
+    config_files = ["generation_config.json", "lit_config.json", "tokenizer_config.json"]
```

Comment:
One could argue that `tokenizer_config.json` is meant to be part of `tokenizer_files`